frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -1,11 +1,21 @@
 [
+  {
+    "type": "dev-console.add/action-group",
+    "properties": {
+      "id": "serverless",
+      "name": "%knative-plugin~Serverless%",
+      "insertBefore": "local-machine",
+      "insertAfter": "container-images"
+    }
+  },
   {
     "type": "dev-console.add/action",
     "flags": {
       "required": ["KNATIVE_EVENTING"]
     },
     "properties": {
       "id": "knative-event-source",
+      "groupId": ["serverless", "developer-catalog"],
```

Comment:
Okay this is the reason to have an array here. We should check this with PM/UX if we want show actions in two different groups.